File: .vscode/C/Estrutura Sequencial/Consumo.c
#include <stdio.h>
#include <string.h>



int main ()
{

    double combustivel, consumo;
    int distancia;

    printf ("Distancia percorrida: ");
    scanf ("%d", &distancia);
    printf ("Combustivel gasto: ");
    scanf ("%lf", &combustivel);

    consumo = (double)distancia / combustivel;

    printf ("Consumo medio = %.3lf\n", consumo);




    return 0;

}

File: Java/Estrutura Sequencial/Consumo.java
import java.util.Locale;
import java.util.Scanner;

public class Consumo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double combustivel, consumo;
        int distancia;

        System.out.print("Distancia percorrida: ");
        distancia = sc.nextInt();
        System.out.print("Combustivel gasto: ");
        combustivel = sc.nextDouble();

        consumo = (double) distancia / combustivel;

        System.out.println("Consumo medio = " + String.format("%.3f", consumo));

        sc.close();

    }
}

